auth/store.go

```diff
@@ -997,18 +997,23 @@ func (as *authStore) AuthInfoFromCtx(ctx context.Context) (*AuthInfo, error) {
 		return nil, nil
 	}
 
-	ts, tok := md["authorization"]
-	if !tok {
-		return nil, nil
-	}
-
-	token := ts[0]
-	authInfo, uok := as.authInfoFromToken(ctx, token)
-	if !uok {
-		plog.Warningf("invalid auth token: %s", token)
-		return nil, ErrInvalidAuthToken
+	for k := range md {
+		if k == "token" || k == "authorization" {
```

Comment:
Yes, I don't think unifying the key names immediately. Just putting a comment or opening an issue for a remainder is ok.